src/crimson/net/ProtocolV2.cc

```diff
@@ -179,6 +181,38 @@ seastar::future<> ProtocolV2::write_flush(bufferlist&& buf)
   return socket->write_flush(std::move(buf));
 }
 
+seastar::temporary_buffer<char>
+ProtocolV2::create(seastar::compat::polymorphic_allocator<char>* const allocator)
+{
+#if 0
+  // space for segments_n, epilogue_n and preable_n+1
+  return seastar::make_temporary_buffer<char>(allocator, 8192);
+#else
+
+  if (!last_returned.empty()) {
```

Comment:
Well, even while using the loopback interface and `rados bench` (fast load generator) the output bufferlist was fairly fragmented (see https://gist.github.com/rzarzynski/38dbc7bb8a25b4136de7ffbdfe1e7dde). On slower interfaces/clients this might be even worse, I'm afraid.